src/interpreter/interpreter.cpp

```diff
@@ -60,8 +65,16 @@ struct ExpressionInterpreter : OverriddenVisitor<ExpressionInterpreter, Flow> {
   Flow visitCallIndirect(CallIndirect* curr) { WASM_UNREACHABLE("TODO"); }
   Flow visitLocalGet(LocalGet* curr) { WASM_UNREACHABLE("TODO"); }
   Flow visitLocalSet(LocalSet* curr) { WASM_UNREACHABLE("TODO"); }
-  Flow visitGlobalGet(GlobalGet* curr) { WASM_UNREACHABLE("TODO"); }
-  Flow visitGlobalSet(GlobalSet* curr) { WASM_UNREACHABLE("TODO"); }
+  Flow visitGlobalGet(GlobalGet* curr) {
+    push(frame().instance.globalValues[curr->name]);
```

Comment:
So simple 🤩 We could even have an `instance()` helper to make this even shorter.

Reply:
done